### PR TITLE
Design cloud shell sidebar

### PR DESCRIPTION
**Pull request by dherault**
## Summary

Design the Cloud shell sidebar.
- 3 steps demo
- Layout
- New theme selector

Left to do:
- Finish theme selector
- Complete information modal
- Create CLI Cheatsheet

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/src/components/shell/TerminalInformation.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        Cloud shell info
+      </Button>
+      <Modal
+        open={open}
+        onClose={() => setOpen(false)}
+      >
+        <ModalHeader onClose={() => setOpen(false)}>
+          Cloud shell info
+        </ModalHeader>
+        Foo
```
Comment (michaeljguarino) [inline www/src/components/shell/TerminalInformation.js:29]: I'm guessing this shouldn't be Foo?

File: www/src/components/shell/onboarding/SplashToLogoTransition.js [hunk 2]
```diff
@@ -129,13 +135,17 @@ function LogoOnly({ children }) {
...
   const [splashTimerDone, setSplashTimerDone] = useState(false)
-  
+
   useEffect(() => {
-    setTimeout(() => {
+    const timeoutId = setTimeout(() => {
       setSplashTimerDone(true)
     }, splashTimeout)
+
+    return () => {
```
Comment (michaeljguarino) [inline www/src/components/shell/onboarding/SplashToLogoTransition.js:144]: can't this be one-lined: `() => clearTimoeut(timeoutId)`

File: www/src/components/shell/TerminalThemeSelector.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  return (
+    <ExtendTheme theme={extendedHonorableTheme}>
+      <DropdownButton
+        label="Change theme"
+        onChange={handleThemeChange}
+      >
+        <Div>
+          input here
```
Comment (michaeljguarino) [inline www/src/components/shell/TerminalThemeSelector.js:48]: ?

File: www/src/components/shell/onboarding/SplashToLogoTransition.js (hunk 2 above)
Comment (dherault) [reply www/src/components/shell/onboarding/SplashToLogoTransition.js:144]: yes it can!

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**